dev/deps/spark-deps-hadoop-3.2-hive-2.3

```diff
@@ -183,7 +183,6 @@ metrics-jmx/4.1.1//metrics-jmx-4.1.1.jar
 metrics-json/4.1.1//metrics-json-4.1.1.jar
 metrics-jvm/4.1.1//metrics-jvm-4.1.1.jar
 minlog/1.3.0//minlog-1.3.0.jar
-mssql-jdbc/6.2.1.jre7//mssql-jdbc-6.2.1.jre7.jar
```

Comment:
Oh, interesting. I didn't notice this.
@gaborgsomogyi . This is worth for minimizing Apache Spark 3.0.0 distribution.
Since this PR cannot land on `branch-3.0`, could you make another PR for this dependency clean-up?

Reply:
Do you know what was the root cause of this dependency?

Reply:
I've just double checked and it has been added by a guy at 24 Nov 2019 [here](https://github.com/apache/spark/blob/a1706e2fa745a8d4aebeebd1663366e13ba80c8c/dev/deps/spark-deps-hadoop-3.2-hive-2.3) :)
You can ask him what was the reason :D

Reply:
> could you make another PR for this dependency clean-up?

Sure. Creating a jira + PR...

Reply:
Filed https://issues.apache.org/jira/browse/SPARK-31821

Reply:
Thanks @dongjoon-hyun Going to roll back the dep remove here as soon as SPARK-31821 is merged.

Reply:
I know you are joking. ;) But, that is an auto-generated file.
> I've just double checked and it has been added by a guy at 24 Nov 2019

Reply:
The PR failed at https://github.com/apache/spark/pull/28640#issuecomment-633919385 .

Reply:
> But, that is an auto-generated file.

Now I see :) Just fixed the other PR.

Reply:
Merged master into this branch to eliminate the deps change. Thanks @dongjoon-hyun